beacon_node/store/src/hot_cold_store.rs

```diff
@@ -1667,6 +1709,102 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> HotColdDB<E, Hot, Cold>
         );
         Ok(())
     }
+
+    /// Try to prune blobs older than the data availability boundary.
+    pub fn try_prune_blobs(&self, force: bool) -> Result<(), Error> {
+        let blob_info = match self.get_blob_info() {
+            Some(old_blob_info) => old_blob_info,
+            None => {
+                return Ok(());
+            }
+        };
+
+        if !force {
+            let epochs_per_blob_prune =
+                Epoch::new(self.get_config().epochs_per_blob_prune * E::slots_per_epoch());
```

Comment:
I think we don't need the multiplication by `slots_per_epoch` as `epochs_per_blob_prune` is already in epoch units

Reply:
`epochs_per_blob_prune` is `u64` but I can convert it to Epochs once upon config creation from cli args

Reply:
Oh then I have misinterpreted the `Epoch::new` method. Change the param name form `slot` to `epoch` in that case?  Seems like a typo almost.

Reply:
Ahhh good catch! That's because both Slot::new and Epoch::new are generated with a macro. The parameter should probably be called something neutral like `n`